### PR TITLE
fixes to TS WF 

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
Adds additional information to Workflow parameters, Return values, Task Queue, and how to handle a query.

### CODE REVIEW

File: docs/typescript/how-to-define-workflow-parameters-in-typescript.md [hunk 1]
```diff
@@ -9,27 +9,29 @@ tags:
   - typescript
 ---
 
-A Task Queue is a dynamic queue in Temporal polled by one or more Workers.
+You can define and pass parameters in your Workflow. In this example, you'll define your arguments in your `client.ts` file and pass those parameters to `workflow.ts` through your Workflow function.
```
Comment (djmagee) [inline docs/typescript/how-to-define-workflow-parameters-in-typescript.md:12]:
```suggestion
You can define and pass parameters in your Workflow. In this example, you define your arguments in your `client.ts` file and pass those parameters to `workflow.ts` through your Workflow function.
```

File: docs/typescript/how-to-define-workflow-parameters-in-typescript.md [hunk 2]
```diff
@@ -9,27 +9,29 @@ tags:
...
 
-When scheduling a Workflow, a `taskQueue` must be specified.
+Start a Workflow with the give parameters in the `client.ts` file. In this example we use set the `name` parameter to `Temporal` and `born` to `2019`. Then set the Task Queue and Workflow Id.
```
Comment (djmagee) [inline docs/typescript/how-to-define-workflow-parameters-in-typescript.md:14]:
```suggestion
Start a Workflow with the parameters that are in the `client.ts` file. In this example we set the `name` parameter to `Temporal` and `born` to `2019`. Then set the Task Queue and Workflow Id.
```
I'm kinda guessing on my edit to the first sentence.

File: docs/typescript/how-to-define-workflow-parameters-in-typescript.md [hunk 3]
```diff
@@ -9,27 +9,29 @@ tags:
...
-const connection = new Connection();
-const client = new WorkflowClient();
-const result = await client.execute(myWorkflow, {
-  taskQueue: 'your-task-queue', // required
-  workflowId: 'your-workflow-id', // required
+import { example } from ‘./workflows’;
+await client.start(example, {
+  args: [{ name: ‘Temporal’, born: 2019 }],
+  taskQueue: ‘my-queue’,
+  workflowId: ‘business-meaningful-id’,
```
Comment (djmagee) [inline docs/typescript/how-to-define-workflow-parameters-in-typescript.md:23]:
```suggestion
import { example } from './workflows';
await client.start(example, {
  args: [{ name: 'Temporal', born: 2019 }],
  taskQueue: 'my-queue',
  workflowId: 'business-meaningful-id',
```

File: docs/typescript/how-to-define-workflow-parameters-in-typescript.md [hunk 4]
```diff
@@ -9,27 +9,29 @@ tags:
...
 });
 ```
 
-When creating a Worker, you must pass the `taskQueue` option to the `Worker.create()` function.
+In `workflows.ts` define the type of the parameter that the Workflow function takes in. The `interface` `ExampleParam` is a name we can now use to describe the requirement in the previous example. It still represents having the two properties called `name` and `born` that is of the type `string`. Then define a function that takes in ta parameter of the type `ExampleParam` and return a `Promise<string>`. The `Promise` object represents the eventual completion, or failure, of `await client.start()` and its resulting value.
```
Comment (djmagee) [inline docs/typescript/how-to-define-workflow-parameters-in-typescript.md:27]:
```suggestion
In `workflows.ts` define the type of the parameter that the Workflow function takes in. The interface `ExampleParam` is a name we can now use to describe the requirement in the previous example. It still represents having the two properties called `name` and `born` that is of the type `string`. Then define a function that takes in a parameter of the type `ExampleParam` and return a `Promise<string>`. The `Promise` object represents the eventual completion, or failure, of `await client.start()` and its resulting value.
```

File: docs/typescript/how-to-define-workflow-parameters-in-typescript.md [hunk 5]
```diff
@@ -9,27 +9,29 @@ tags:
...
 
 ```typescript
-import { Connection, WorkflowClient } from '@temporalio/client';
-const connection = new Connection();
-const client = new WorkflowClient();
-const result = await client.execute(myWorkflow, {
-  taskQueue: 'your-task-queue', // required
-  workflowId: 'your-workflow-id', // required
+import { example } from './workflows';
+await client.start(example, {
```
Comment (lorensr) [inline docs/typescript/how-to-define-workflow-parameters-in-typescript.md:20]:
```suggestion

...
await client.start(example, {
```

File: docs/typescript/how-to-handle-a-query-in-a-workflow-in-typescript.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+Query Handlers can return values inside a Workflow in TypeScript.
+
+You make a Query with `handle.query(query, ...args)`. A Query needs a return value, but can also take arguments.
+
+```typescript
+import * as wf from '@temporalio/workflow';
+
+function useState<T = any>(name: string, initialValue: T) {
```
Comment (lorensr) [inline docs/typescript/how-to-handle-a-query-in-a-workflow-in-typescript.md:18]: Not a good example. Here's one:

https://docs.temporal.io/typescript/workflows#define-signals-and-queries-statically

**Review by lorensr — APPROVED** [review]

**Review by djmagee — APPROVED** [review]

**Pull request merged**